content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -246,10 +248,33 @@ message ContainerDevices {
     TopologyInfo topology = 3;
 }
 ```
+``{{< note >}}
```

Comment:
nit: Do you need the two quotes before the note callout?

Reply:
@kbhawkey Removed, Thanks